examples/sampling_c3/urdf/frame/frame_controller.sdf

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<sdf version="1.7">
+  <model name="anything">    
+    <link name="body">
+      <inertial>
+        <pose>0 0 0 0 0 0</pose>
+        <mass>1</mass>
+        <!-- Inertia values were calculated to be consistent with the mass and
+             geometry size, assuming a uniform density. -->
```

Comment:
These comments should be removed since they are no longer true -- ensure the autogeneration script also doesn't insert these into URDFs it writes.